qa/tasks/cephfs/mount.py

```diff
@@ -695,7 +695,7 @@ def run_python(self, pyscript, py_version='python3', sudo=False):
         p.wait()
         return p.stdout.getvalue().strip()
 
-    def run_shell(self, args, timeout=900, **kwargs):
+    def run_shell(self, args, timeout=300, **kwargs):
```

Comment:
@cephfs @batrick This change is not strictly related to this PR/commit. Is it okay to keep it here? I often am told to create a separate PR but this change is very very minor.

Reply:
Why is "300" more reasonable than "900"? What problem is being solved?

Reply:
(yes, a separate PR is better)

Reply:
Done